tests/test_core.py

```diff
@@ -59,6 +59,18 @@ def decorated(self):
   return decorated
 
 
+def bleeding_edge_wasm_backend(f):
+  def decorated(self):
+    if not self.is_wasm_backend():
+      self.skipTest('only works in wasm backend')
+    if not V8_ENGINE or V8_ENGINE not in JS_ENGINES:
+      self.skipTest('only works in d8 for now')
+    if self.is_wasm_backend() and not self.get_setting('WASM'):
+      self.skipTest('wasm2js only supports MVP for now')
+    f(self, js_engines=[V8_ENGINE])
```

Comment:
Do we need to append feature flags to `V8_ENGINE` to support the bleeding edge features, or do we assume the user has already done that in their config?

Reply:
I added them to emcc_args manually in the test. Not sure if there's a nicer way? Maybe a parameter to the decorator? Not sure that's clearer.

Reply:
Yeah actually a decorator parameter that can take flags like `--experimental-wasm-simd` would be good, so you can say just which part of the bleeding edge you care about. I guess this particular test doesn't need that, though.